packages/orbit-components/src/CarrierLogo/index.tsx

```diff
@@ -76,18 +85,23 @@ StyledImage.defaultProps = {
   theme: defaultTheme,
 };
 
-export const StyledCarrierLogo = styled.div<{ theme: Theme; carriers: Carrier[]; size: Size }>`
-  ${({ theme, carriers, size }) => css`
+export const StyledCarrierLogo = styled.div<{
+  theme: Theme;
```

Comment:
I think we can omit it here and just do:
```suggestion
  theme: typeof defaultTheme;
```